src/unitxt/operators.py

```diff
@@ -1215,30 +1216,48 @@ def _is_required(self, instance: dict) -> bool:
         return True
 
 
-class FilterByQuery(SingleStreamOperator):
+class ComputeExpressionMixin(Artifact):
+    """Computes an expression expressed over fields of an instance.
+
+    Args:
+        expression (str): the expression, in terms of names of fields of an instance
+        imports_list (List[str]): list of names of imports needed for the evaluation of the expression
+    """
+
+    expression: str
+    imports_list: List[str] = []
```

Comment:
This is not good. as the imports_list here is assigned at the class level which means all the instances will forver use the same list

Can be fixed by first: `from .dataclass import OptionalField`

Then `imports_list: List[str] = OptionalField(default_factory=list)`